flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolImpl.java

```diff
@@ -469,6 +469,47 @@ public void releaseSlot(@Nonnull SlotRequestId slotRequestId, @Nullable Throwabl
 			.collect(Collectors.toList());
 	}
 
+	@Override
+	public boolean isSlotRequestBulkFulfillable(final PhysicalSlotRequestBulk slotRequestBulk) {
```

Comment:
This sounds good to me.
I had wanted to do it like that but encountered some problems. I do not remember the problems unfortunately.
I will try and see if it is still a problem. Maybe not since the design was adjusted during  developing.

Reply:
done.